config.toml

```diff
@@ -13,7 +13,7 @@ home = [ "HTML", "RSS", "JSON"]
   author = "Kubeshop" # add your company name
   github = "https://github.com/kubeshop/botkube" # add your github profile name
   email = "frontdesk@kubeshop.io"
-  description = "BotKube is a messaging bot for monitoring and debugging Kubernetes clusters."
```

Comment:
Didn't do the review, just noticed this. I think this file should be removed as it is still from the Hugo config, right?

Reply:
As far as I know this is config for GitLab runner. I can remove it because we use GitHub Action and probably it's a left overs from old version.

Reply:
This is a config for Hugo (you can see e.g. `theme = "hugo-theme-learn"` property). Yes, please remove it 👍